distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -22,25 +24,25 @@ polyglot java import org.enso.base.Regex_Utils
 
 type Filter_Condition
     ## Is less than a value (or another column, in case of Table operations)?
-    Less than:Any action:Filter_Action=Filter_Action.Keep
+    Less than=(Missing_Argument.throw "than") action:Filter_Action=Filter_Action.Keep
```

Comment:
- Is this still needed after merge of #10187 ?

Reply:
Will try this in the next PR and if it works switch to then.

Reply:
Hello @jdunkerley, does the
- #10187 

approach works for you? Can it be used instead of `Missing_Argument.throw "than", please? CCing @GregoryTravis

Reply:
I think this is the best solution now because it requires no changes to Numbers.enso type signatures. It adds a special case to the error class but that seems acceptable.